Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.14 as default
+FROM alpine:3 as default
```

Comment:
This looks fine as it will always pull in the latest version of 3.x, but my only concern is it's difficult to tell what version of Alpine was used in the build. As long as we can keep a receipt of that, and we're okay with always rolling forward, then this should be fine.